eventhandler.py

```diff
@@ -24,8 +25,14 @@ def on_new_comment(self, api, payload):
     def on_pr_closed(self, api, payload):
         pass
 
+    def on_build_status(self, api, payload):
+        pass
+
     def handle_payload(self, api, payload):
-        action = payload['action']
+        if 'context' in payload:
```

Comment:
Instead of searching for "context" in payload, we should probably try accessing `payload['context']` and catch the raised `KeyError`